texastribune/donations
# Move from Yarn to NPM

#### What's this PR do?
What the title says.

Also specifies a Node version in `package.json` so we can be more explicit with the Heroku build pack.

#### Why are we doing this? How does it help us?
+ The upgrade to Yarn 2 [sounds somewhat brutal](https://dev.to/arcanis/introducing-yarn-2-4eh1#what-will-happen-to-the-legacy-codebase)
+ NPM now offers a lock file. It not having one is why we switched to Yarn.
+ Data viz uses NPM, so it's good to be consistent
+ NPM just feels more like the industry standard
+ I don't want to use more FB tools than we have to

#### How should this be manually tested?
+ `make`
+ Try all the scripts in `package.json` and verify things look OK. In particular, do `make restart` and view `/account/`, `/donate` and `/circle` and confirm things look good.
+ View this PR's Heroku URL and confirm stuff is looking OK there, too (a sanity check that Heroku is dealing with NPM as expected)

#### How should this change be communicated to end users?
NA.

#### Are there any smells or added technical debt to note?
NA.

#### What are the relevant tickets?
Sort of this:
https://3.basecamp.com/3098728/buckets/736178/todos/2380825307

#### Have you done the following, if applicable:
* [x] Checked for security implications? *(I updated one vulnerability that NPM flagged)*
* [x] Updated the documentation/wiki? *(Updated README)*
